pdaltools/add_points_in_pointcloud.py

```diff
@@ -35,37 +37,80 @@ def parse_args(argv=None):
         default=1000,
         help="width of tiles in meters",
     )
+    parser.add_argument(
+        "--spacing",
+        type=float,
+        default=0,
+        help="spacing between generated points in meters",
+    )
+    parser.add_argument(
+        "--altitude_column",
+        "-z",
+        type=str,
+        required=True,
+        default="RecupZ",
+        help="altitude column name from input geometry",
+    )
 
     return parser.parse_args(argv)
 
 
-def clip_3d_points_to_tile(input_points: str, input_las: str, crs: str, tile_width: int) -> gpd.GeoDataFrame:
+def clip_3d_lines_to_tile(
+    input_lines: gpd.GeoDataFrame, input_las: str, crs: str, tile_width: int
+) -> gpd.GeoDataFrame:
     """
-    Add points from a GeoJSON file in the LIDAR's tile.
+    Clip lines from a GeoDataFrame to the LIDAR's tile.
```

Comment:
Petit pinaillage : en fait c'est pas un vrai clip, vu qu'on garde les lignes entières même si elles sont pas complètement dans la dalle (d'ailleurs c'est bien ce que tu fais dans cette fonction) :)

Proposition :
```suggestion
    Select lines from a GeoDataFrame that intersect the the LIDAR tile.
```